src/pyeval.jl

```diff
@@ -202,32 +202,37 @@ pasted into the Python code.   This allows you to evaluate code
 where the code itself is generated by a Julia expression.
 """
 macro py_str(code, options...)
+    m = :(pynamespace($__module__))
+    esc(macroexpand(__module__, :(PyCall.@_py_str($m, $m, $code, $(options...)))))
```

Comment:
If it were a macro, I don't think we need to call `macroexpand` manually here?

```suggestion
    esc(:($PyCall.@_py_str($m, $m, $code, $(options...))))
```